cli/src/forge.rs

```diff
@@ -27,7 +28,39 @@ mod utils;
 #[tracing::instrument(err)]
 fn main() -> eyre::Result<()> {
     utils::subscriber();
-
+    if cfg!(target_os = "windows") {
+        println!("Can't source env files on windows");
+    } else {
+        let home: String = env::var("HOME")?;
```

Comment:
there is the https://crates.io/crates/dirs crate that has `dirs::home_dir()`

Reply:
yy, I generally try to follow a minimal crate approach. Do you think that using the HOME env variable has gotchas (for UNIX systems that is). 

For windows, I am not sure we even support the platform, that's why I made the distinction.